config.cmake.h.in

```diff
@@ -0,0 +1,173 @@
+/* major version */
```

Comment:
Do we need both this file and `fplll.cmake.pc.in`? It looks auto-generated.

Reply:
i was just following `config.h.in` and `fplll.pc.in` in the original setup and use the simplest `configure_file` instead of anything fancier that creates this file (as configure_file only copy over and fill in variables)

happy to see less duplication